src/components/application_manager/src/commands/hmi/get_urls.cc

```diff
@@ -194,15 +194,15 @@ void GetUrls::ProcessPolicyServiceURLs(const policy::EndpointUrls& endpoints) {
   object[msg_params].erase(hmi_request::service);
   object[msg_params][hmi_response::urls] = SmartObject(SmartType_Array);
   SmartObject& urls = object[msg_params][hmi_response::urls];
-  const std::string mobile_app_id = app->policy_app_id();
+  const std::string mobile_app_policy_id = app->policy_app_id();
```

Comment:
@ZhdanovP Keep consistency. Somewhere you call such variable as `mobile_app_policy_id`, afterwards - `app_policy_id`